src/applications/vaos/appointment-list/components/AppointmentsPageV2/AppointmentListItem.jsx

```diff
@@ -72,26 +73,30 @@ export default function AppointmentListItem({ appointment, facility }) {
   const link = isCommunityCare
     ? `cc/${appointment.id}`
     : `va/${appointment.id}`;
+  const idClickable = `id-${appointment.id}`;
 
   return (
     <li
+      id={idClickable}
       data-request-id={appointment.id}
-      className="vaos-appts__card vaos-appts__card--clickable"
+      className="vaos-appts__card--clickable vads-u-margin-bottom--3"
       data-cy="appointment-list-item"
     >
       <div
-        className="vads-u-padding--2 medium-screen:vads-u-padding--3 medium-screen:vads-u-margin-bottom--3 vads-u-display--flex vads-u-align-items--center"
+        className="vads-u-padding--2 vads-u-display--flex vads-u-align-items--left vads-u-flex-direction--column medium-screen:vads-u-padding--3 medium-screen:vads-u-flex-direction--row medium-screen:vads-u-align-items--center"
         onClick={() =>
-          !window.getSelection().toString() ? history.push(link) : null
+          !window.getSelection().toString()
+            ? (focusElement(`#${idClickable}`), history.push(link))
```

Comment:
I could see a screen reader doing something weird with this extra focus, but VoiceOver seems fine with it, so hopefully the rest are as well.

Reply:
Done